python/rascaline/examples/splined-radial-integral.py

```diff
@@ -114,16 +117,16 @@ def laplacian_eigenstate_basis_derivative(n, el, r):
 # %%
 #
 # The radial basis functions and their derivatives can be input into a spline
-# generator. This will output the positions of the spline points, the
+# generator class. This will output the positions of the spline points, the
 # values of the basis functions evaluated at the spline points, and the
 # corresponding derivatives.
-spline_points = generate_splines(
-    laplacian_eigenstate_basis,
-    laplacian_eigenstate_basis_derivative,
-    max_radial,
-    max_angular,
-    cutoff,
-    requested_accuracy=1e-5,
+spliner = SplinerFromFunction(
+    radial_integral=laplacian_eigenstate_basis,
+    spline_cutoff=cutoff,
+    max_radial=max_radial,
+    max_angular=max_angular,
+    radial_integral_derivative=laplacian_eigenstate_basis_derivative,
```

Comment:
```suggestion
    radial_integral=laplacian_eigenstate_basis,
    radial_integral_derivative=laplacian_eigenstate_basis_derivative,
    spline_cutoff=cutoff,
    max_radial=max_radial,
    max_angular=max_angular,
```

this feels like a more logical grouping to me